api/client/volume/client.go

```diff
@@ -658,3 +660,23 @@ func (v *volumeClient) CloudBackupSchedEnumerate() (*api.CloudBackupSchedEnumera
 	}
 	return enumerateResponse, nil
 }
+
+func (v *volumeClient) SnapshotGroup(groupID string, labels map[string]string) (*api.GroupSnapCreateResponse, error) {
+
+	response := &api.GroupSnapCreateResponse{}
+	request := &api.GroupSnapCreateRequest{
+		Id:     groupID,
+		Labels: labels,
+	}
+
+	req := v.c.Post().Resource(snapPath + "/groupsnap").Body(request)
```

Comment:
nit: call it /snapshotgroup (so its consistent with api ?)

Reply:
surething